homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -1166,25 +1166,19 @@ async def async_set_dry_off(self):
 class XiaomiAirHumidifierMiot(XiaomiAirHumidifier):
     """Representation of a Xiaomi Air Humidifier (MiOT protocol)."""
 
+    mode_mapping = {
```

Comment:
Write this attribute in all capital letters so its clear that it's considered a constant that shouldn't change.